qcengine/util.py

```diff
@@ -474,3 +473,19 @@ def which(command, return_bool=False):
         return bool(ans)
     else:
         return ans
+
+
+def safe_version(*args, **kwargs):
+    """
+    Package resources is a very slow load
+    """
+    import pkg_resources
```

Comment:
`pkg_resources` has widespread use. It's not really avoidable in an engine run, right? if so, not following how this is better than an outright import

Reply:
that is, I renovated found and get_version for dftd3 and psi4 as a model, but rdkit and torchani should probably follow suit.

Reply:
It will be used, but doesn't take an extra `0.4` seconds at init time. For example this will slow down QCFractal import by 0.4 seconds even it is never used.

Reply:
hmpf, that is slow